drivers/python/README.md

```diff
@@ -1,3 +1,40 @@
 # Python Driver
 
 For more information about this driver see the [Python API](http://www.rethinkdb.com/api/python/) page.
+
+# Contributing
+
+## Installing
+If you would like to start helping develop the python driver you need to do
+the following:
+
+```bash
+git clone https://github.com/rethinkdb/rethinkdb.git
+./configure --allow-fetch
+cd drivers/python
+make
+pip install -e .
+```
+
+The `make` command will take awhile, if you have a lot of cores you can speed
```

Comment:
That being said, the tests will require the full server. So I guess there should be an extra step about running `make` in the rethinkdb root directory?